torch/functional.py

```diff
@@ -1,5 +1,5 @@
 from typing import (
-    Tuple, Optional, Union, Any, Sequence, TYPE_CHECKING
+    List, Tuple, Optional, Union, Any, Sequence, TYPE_CHECKING
```

Comment:
Is this the same as the one from _jit_internal?

Reply:
Yes, the only `List` in that file is imported from `typing`: https://github.com/pytorch/pytorch/blob/v1.11.0/torch/_jit_internal.py#L29